Samples/Samples/App.xaml

```diff
@@ -11,19 +11,29 @@
                 <Setter Property="Visual" Value="{x:Static local:App.PreferredVisual}" />
             </Style>
             <Style TargetType="Editor">
-                <Setter Property="TextColor" Value="{OnPlatform iOS='Black'}" />
+                <Setter Property="TextColor" Value="Black" />
+                <Setter Property="Visual" Value="Default" />
```

Comment:
I see you are resetting the visual. Do the colors not fix it?

Reply:
Visual Material is not working very well. I don't have time to play and configure it in a sample project. I just want to be able to test Essentials API

Reply:
I do not see a problem in the design of this application, the main thing is that it is comfortable for us